src/makeRobot.js

```diff
@@ -38,7 +38,72 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
```

Comment:
```suggestion
  return {
```
you can just return immediately